config/settings/base.py

```diff
@@ -556,19 +560,45 @@
 # Configuration used for generating Celery tasks for the asynchronous sprint process. These tasks are created
 # or updated (if they already exist) while ending a sprint, so any manual changes to them will be lost.
 SPRINT_ASYNC_TASKS = {
+    # FIXME: The sprint completion is not automated yet, and this should be done only if the sprints all cells are
+    #  completed for all cells. Add this to the sprint completion pipeline, once it's automated.
+    "sprints.dashboard.tasks.create_estimation_session_task": {
+        "name": "[ASYNC] Close estimation session",
```

Comment:
@Agrendalath, shouldn't this be `Create estimation session`?

Reply:
@0x29a, yep, good catch. I addressed this in #70.